apps/web/components/imports/schema/ColumnsTable.tsx

```diff
@@ -14,20 +14,40 @@ import { EditIcon } from '@assets/icons/Edit.icon';
 import { CloseIcon } from '@assets/icons/Close.icon';
 import { CheckIcon } from '@assets/icons/Check.icon';
 import { DeleteIcon } from '@assets/icons/Delete.icon';
+import { MultiSelect } from '@mantine/core';
 
 interface ColumnsTableProps {
   templateId: string;
 }
 
 export function ColumnsTable({ templateId }: ColumnsTableProps) {
   const [showAddRow, setShowAddRow] = useState(false);
-  const { register, columns, control, handleSubmit, onEditColumnClick, onDeleteColumnClick, isColumnCreateLoading } =
-    useSchema({
-      templateId,
-    });
+  const [isFocuedOnSelect, setIsFocusOnSelect] = useState(false);
+
+  const {
+    register,
+    watch,
+    columns,
+    control,
+    handleSubmit,
+    onEditColumnClick,
+    onDeleteColumnClick,
+    isColumnCreateLoading,
+    formState: { errors },
+  } = useSchema({
+    templateId,
+  });
+
+  const typeValue = watch('type');
 
   return (
-    <form onSubmit={handleSubmit}>
+    <form
+      onSubmit={(e) => {
+        e.preventDefault();
+        handleSubmit();
+        setIsFocusOnSelect(false);
+      }}
```

Comment:
Can we try using `useRef` for `isSelectFocused` and bringing `setIsFocusSelect` = `false` in handleSubmit code itselft?

Reply:
tried to implement but i throw not assignable error